docs/validationtest/README.md

```diff
@@ -1,8 +1,19 @@
-# Subscriber
+# Validationtest
 
-This tool is used to create Kyma subscriptions and their corresponding subscribers.
+This set of tools is used to publish continues legacy and cloudevents the Eventing publisher proxy Kyma. In addition to the publisher
+a set of subscribers is created with corresponding subscriptions.
 
-## Eventing infrastructure
+## Application to event type mapping
+
+Currently, it is hardcoded which application sends which event type as follows:
+
+| Application   | Event type                                       |
+|---------------|--------------------------------------------------|
+| commerce      | order.created                                    |
+| appname       | DocuSing_BO.Account_DocuSign.Updated             |
+| no-app        | New.Some-Other.Order-äöüÄÖÜβ.Final.C-r-e-a-t-e-d |
+
+## Subscription
 
```

Comment:
Can we add an introductory sentence to this list?

Reply:
i tried ;)

Reply:
Excellent, thank you!